test/interoperability/interop_test_suite.py

```diff
@@ -0,0 +1,537 @@
+        assert (
+            resp.status_code == 200
+        ), f"Failed to find ISA_1 in Primary DSS: {primary_dss}"
+        data = resp.json()["service_area"]
+
+        time_end = datetime.datetime.utcnow() + datetime.timedelta(
+            seconds=SHORT_WAIT_SEC
+        )
+        put_resp = dss_map[primary_dss].put(
+            f"/identification_service_areas/{isa_1_uuid}/{data['version']}",
+            json={
+                "extents": {
+                    "spatial_volume": {
+                        "footprint": {"vertices": VERTICES},
+                        "altitude_lo": 20,
+                        "altitude_hi": 400,
+                    },
+                    "time_start": data["time_start"],
+                    "time_end": time_end.strftime(DATE_FORMAT),
+                },
+                "flights_url": "https://example.com/dss",
+            },
+        )
+        assert put_resp.status_code == 200, f"Failed to update ISA_1 with new end time"
+
+        updated_data = put_resp.json()
+        assert updated_data["service_area"]["time_end"] == time_end.strftime(
+            DATE_FORMAT
+        ), f"Unsuccessful Update; no change to end time"
+
+    def testStep6(
+        self, dss_map: Dict[str, clients.DSSClient], primary_dss: str, **kwargs
+    ) -> None:
+        """Can delete all Subscription in primary DSS"""
+        dss = dss_map[primary_dss]
+        for name, context in self.context.items():
+            if name.startswith("sub_1_"):
+                resp = dss.delete(f"/subscriptions/{context.key}/")
+                assert (
+                    resp.status_code == 200
+                ), f"Failed to delete {context.key} from Primary DSS: {primary_dss}"
+
+    def testStep7(
+        self,
+        dss_map: Dict[str, clients.DSSClient],
+        primary_dss: str,
+        all_other_dss: List[str],
+    ) -> None:
+        """Subscription deletion from ID index was effective on primary DSS"""
+        all_dss = [primary_dss] + all_other_dss
+        for index, dss_name in enumerate(all_dss):
+            dss = dss_map[dss_name]
+            sub_uuid = self.context[f"sub_1_{index}_uuid"].key
+            resp = dss.get(f"/subscriptions/{sub_uuid}")
+            assert (
+                resp.status_code == 404
+            ), f"Expecting code 404, found {resp.status_code}"
+
+    def testStep8(
+        self,
+        dss_map: Dict[str, clients.DSSClient],
+        primary_dss: str,
+        all_other_dss: List[str],
+    ) -> None:
+        """Subscription deletion from geographic index was effective on primary DSS"""
+        all_dss = [primary_dss] + all_other_dss
+        for dss in all_dss:
+            resp = dss_map[dss].get(f"/subscriptions?area={GEO_POLYGON_STRING}")
+            assert (
+                resp.status_code == 200
+            ), f"Expecting code 200, found {resp.status_code}"
+
+            data = resp.json()
+            assert (
+                data["subscriptions"] == []
```

Comment:
To be as robust as possible, we actually only care that none of the known-deleted Subscriptions are included.  There could perhaps be pre-existing Subscriptions.